vaadin-icons-flow-parent/vaadin-icons-flow/src/main/java/com/vaadin/flow/component/icon/IronIcon.java

```diff
@@ -34,6 +34,8 @@
  *             {@link Icon}
  */
 @Tag("iron-icon")
+@NpmPackage(value = "@vaadin/vaadin-icons", version = "21.0.0-alpha6")
+@JsModule("@vaadin/vaadin-icons/iconset.js")
```

Comment:
I wonder if it would be reasonable to only import `@vaadin/vaadin-icons` since it automatically imports `iconset.js` and `iron-icon`, not sure if it is necessary though.

```json
{
  "name": "@vaadin/vaadin-icons",
  "version": "21.0.0-alpha8",
  "description": "Collection of 600+ unique icons designed for web applications",
  "main": "vaadin-icons.js",
  "module": "vaadin-icons.js", <--------------
  "repository": "vaadin/vaadin-icons",
```

Reply:
Feels cleaner to use explicit imports for what is needed by the component instead of relying on some bundle defined by the WC package.

Now that you mentioned, we should probably change the "main" and "module" definitions in package.json since `vaadin-icons.js` is deprecated in favor of `vaadin-iconset.js`.

Reply:
Yeah, feels cleaner to me as well 👍  especially if the `vaadin-icons.js` is deprecated.